ui/shared/components/panel/variants/Annotations.jsx

```diff
@@ -332,13 +372,14 @@ const Annotations = React.memo(({ variant, mainGeneId, showMainGene }) => {
         variant.liftedOverPos ? (
           <div>
             hg19:
-            <UcscBrowserLink variant={variant} useLiftover includeEnd={!!variant.svType || !variant.ref} />
+            <UcscBrowserLink variant={variant} useLiftover includeEnd={!!svType || !variant.ref} />
+            {translocLink(liftedTargetLocus, true)}
```

Comment:
similar to above, but we want the second link to  be shown conditonially on `endChrom` and use the `isEnd` parameter alone with the `useLiftover` parameter. We should also update `UcscBrowserLink` to always take the lifted end coordinates from `rg37LocusEnd` if thats in the variant object